explorer/interpreter/resolve_control_flow.cpp

```diff
@@ -138,6 +138,14 @@ static auto ResolveControlFlow(Nonnull<Statement*> statement,
 
 auto ResolveControlFlow(Nonnull<Declaration*> declaration) -> ErrorOr<Success> {
   switch (declaration->kind()) {
+    case DeclarationKind::DestructorDeclaration: {
+      auto& destructor = cast<DestructorDeclaration>(*declaration);
```

Comment:
```suggestion
    case DeclarationKind::DestructorDeclaration:
    case DeclarationKind::FunctionDeclaration: {
      auto& destructor = cast<CallableDeclaration>(*declaration);
```

Can these be combined?